### DISCUSSION TITLE
Упоминание ролей при публикации в Discord

### DISCUSSION THREAD

**Discussion by Bounty38** [MrZillaGold/VK2Discord#319] [Q&A] [answered]
Как сделать упоминание ролей при публикации поста из ВК в Discord?

Например @everyone в начале сообщения.

**Comment by egorprnn** [comment, marked as answer]
Вы можете указывать любой текстовый контент который будет сопровождаться вместе с основным сообщением, за это отвечает поле `content` в объекте `discord` конфига.

https://github.com/MrZillaGold/VK2Discord/blob/master/CONFIG_FIELDS.md#discord

**Reply by Bounty38** [reply]
Спасибо!
